publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -186,6 +179,22 @@ public Resource markPublicationForDeletion(Resource resource)
             .orElseThrow(failure -> handleConditionFailureException(failure, resource));
     }
 
+    private static List<Resource> queryResultToResourceList(QueryResult result) {
+        return result.getItems()
+            .stream()
+            .map(resultValuesMap -> parseAttributeValuesMap(resultValuesMap, ResourceDao.class))
+            .map(ResourceDao::getResource)
+            .collect(Collectors.toList());
+    }
+
+    private Resource fetchEventuallyConsistentResource(Resource newResource) {
```

Comment:
Hmm. In my mind, this action seems like something we shouldn't be doing as the response should be 202 + location header

Reply:
I agree, my intention was to not break the current contract. Maybe we can create a bug issue and fix it later?